.circleci/config.yml

```diff
@@ -175,6 +194,37 @@ workflows:
           requires:
             - run_specs
             - linters
+      - deploy_uat:
+          context: laa-estimate-eligibility-uat
+          requires:
+            - build_and_push_to_ecr
+
+  merge_pr_to_main:
+    jobs:
+      - install_dependencies:
+          filters:
+            branches:
+              only: main
+      - linters:
+          filters:
+            branches:
+              only: main
```

Comment:
Is this redundant given the `requires` having that branch condition too?

Reply:
Possibly. I will test it, I was going with belt and braces approach.

Reply:
Appears I can remove all filters after the first one.